src/main/java/se/kth/sda/skeleton/posts/PostService.java

```diff
@@ -1,7 +1,76 @@
 package se.kth.sda.skeleton.posts;
 
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import se.kth.sda.skeleton.exception.ResourceNotFoundException;
+
+import java.util.Collections;
+import java.util.List;
 
 @Service
 public class PostService {
+
+    PostRepository postRepository;
+
+    @Autowired
+    public PostService(PostRepository postRepository) {
+        this.postRepository = postRepository;
+    }
```

Comment:
Suggested javadoc:

/**
     * Constructs a PostService and automatically assigns its postRepository field
     * 
     * @param postRepository an object that implements interface PostRepository
     */

Reply:
added